.npmrc

```diff
@@ -1 +1 @@
-package-lock=false
+registry=https://registry.npmjs.org
```

Comment:
I would rather drop this since it is default personally. Not super opinionated about it, but we can delete the file and who doesn't like deleting files?

Reply:
I mostly put this here incase those who have private regsitries and use their work machine instead of their own personal machine. This just makes sure we are absolutely pointed to the npmjs registry. If we feel we don't need it I can remove it.

Reply:
Your reasoning makes sense, but home directory `.npmrc` files are an anti-pattern and anyone using a private registry should be strongly suggesting that projects have their own `.npmrc` file which points at their internal registry. Any other configuration is a security risk and causes more friction when on-boarding new engineers.

So yes, personally I would rather not make it easier for folks using home directory configs by adding this override.

Reply:
Notably, the source of packages shouldn't matter so long as we are generating lockfiles (which is what this change aims to do). Integrity values shouldn't change across the registry mirrors/caches & would get caught by any kind of security-specific tooling

Reply:
> Notably, the source of packages shouldn't matter so long as we are generating lockfiles

I think I disagree on this point. I think an organization who runs their own registry typically wants to ensure that *ALL* packages come from that registry. Even if it is the same content with the same integrity values, having server logs for installs is an important factory for incident response. I know for sure I don't want any packages coming from anywhere other than our registry at Netflix.